app/graph/mutations/base_mutation.rb

```diff
@@ -0,0 +1,54 @@
+module Mutations
+  class BaseMutation < ::GraphQL::Schema::RelayClassicMutation
+    class << self
+      private
+
+      def define_shared_bulk_behavior(action, subclass, mutation_target, parents)
+        subclass.graphql_name "#{action.to_s.capitalize}#{mutation_target.camelize.pluralize}"
+
+        subclass.argument :ids, [GraphQL::Types::ID], required: true
+        subclass.field :ids, GraphQL::Types::ID, null: false
```

Comment:
TODO: Double check this, since null: false means the opposite of what I thought (that it's not nullable)